Gho collateral draft (#2113)
Synthetixio/SIPs: content/sips/sip-draft_add_GHO_as_collateral_arbitrum.md

---
sip: <to be assigned>
title: Support GHO as LP collateral on Arbitrum
network: Arbitrum
status: Draft
type: Governance
author: TokenLogic (@TokenLogic-com-au)
implementor: TokenLogic (@TokenLogic-com-au)
release: TBD
implementation-date: TBD
discussions-to: TBD
created: 2024-09-18
---

## Simple Summary

This SIP proposes to add support for Aave's GHO Stablecoin (GHO) as LP collateral on Sinthetix v3 on Arbitrum.

## Abstract

Add GHO as a supported collateral type for Arbitrum V3 deployment with minting at 125% C-ratio and a maximum LP cap of $50M.

## Motivation

GHO is a decentralized multi-collateral stablecoin that is fully backed, transparent and native to the Aave Protocol with over [155M](https://aave.tokenlogic.xyz/gho) of circulating supply.

> The collateral ratio exceeds [2.20](https://aave.tokenlogic.xyz/collateral) and is consists of by high-quality assets like wstETH and wETH making up [23.67%](https://aave.tokenlogic.xyz/gho) and [20.20%](https://aave.tokenlogic.xyz/gho) respectively of all collateral.

Reviewer: ```suggestion
```

The Aave Liquidity Committee (ALC) is eager to explore integrations that expected to drive GHO's expansion across various protocols and chains.

> Further details about GHO can be found [here](https://docs.gho.xyz/).

Reviewer: ```suggestion

```

The Chainlink CCIP bridge was deployed by Aave Governance and has been in production since early July. GHO has also been integrated into Aave v3 on Arbitrum.

> With the initial phase of the launch complete, it's time to accelerate GHO's growth on Arbitrum. Integrating GHO as a collateral asset into Synthetix v3 is a strategic move that supports this expansion.

Reviewer: ```suggestion

```

## Specification

### Overview

> Implementation of this SIP will be contingent on the availability of a high frequency GHO price oracle on Arbitrum. This is actively being developed by Chainlink.

Reviewer: ```suggestion
Implementation of this SIP can happen today through the available GHO Chainlink Data push Feed on Arbitrum. Chainlink Data Streams is also actively developing a GHO low latency pull feed for use on Synthetix markets.
```
